tutorials/desktop/PGP-encryption-from-terminal.md

```diff
@@ -0,0 +1,216 @@
+---
+id: PGP-encryption-from-terminal
+summary: This tutorial will cover installing OpenPGP, creating your keys, file encryption, decryption and signing from the command line.
+categories: desktop
+tags: PGP, encryption, tutorial, terminal, keys
+difficulty: 2
+status: draft
+feedback_url: https://github.com/canonical-websites/tutorials.ubuntu.com/issues
+published:
```

Comment:
I know you don't know when this will actually be published online, but the code that runs the site needs a date in here anyhow, so just add today (we can change it later if we want)

Reply:
If you add a commit to this branch it will update and you should see the CI (travis) checks passing